demo/offline_section.js

```diff
@@ -202,11 +202,20 @@ shakaDemo.storeDeleteAsset_ = function() {
       return shakaDemo.refreshAssetList_();
     });
   } else {
+    let configureCertificate = Promise.resolve();
+
     let asset = shakaDemo.preparePlayer_(option.asset);
+
+    if (asset.certificateUri) {
+      configureCertificate = shakaDemo.requestCertificate_(asset.certificateUri)
+        .then(shakaDemo.configureCertificate_);
+    }
+
+    p = configureCertificate.then(function() {
     let nameField = document.getElementById('offlineName').value;
```

Comment:
This needs to be indented.  Running `./build/all.py` will check for style violations.

Reply:
OK will fix.  
BTW I did run all.py (or check.py) and it does not trap this violation.